appscale/tools/admin_api/version.py

```diff
@@ -104,6 +104,21 @@ def from_yaml(app_yaml):
         except StandardError:
             raise AppEngineConfigException('Invalid app.yaml: automatic_scaling invalid.')
```

Comment:
With these changes, it would be worth revisiting if this `raise` is still needed. In the app.yaml docs, it says `min_instances` and `max_instances` are optional. I'm not sure if GAE will reject an empty `automatic_scaling` section, though.